numba/roc/hsadrv/driver.py

```diff
@@ -20,7 +20,6 @@
 from numba.core import utils, config
 from .error import HsaSupportError, HsaDriverError, HsaApiError
 from numba.roc.hsadrv import enums, enums_ext, drvapi
-from numba.core.utils import longint as long
```

Comment:
Perhaps remove `longint` from `numba.core.utils` too? A number of core devs discussed removal of Py2 things from `utils` the other day, consensus was should be fine to remove.

Reply:
It it worth launching another CI run for that, or should I just make a follow-up PR to remove all the now-unused things in utils?

Reply:
Perhaps the latter if that's ok? There's 3 PRs of similar ilk and it'd probably more efficient doing that. Thanks!

Reply:
Sure - although I can't really make that PR until this and those other 2 PRs are merged

Reply:
Thanks, and no problem, all are approved, aiming to merge them today.